### PR TITLE
Changes needed to output metadata for pegasus

### PR DESCRIPTION
**Pull request by duncan-brown**
This patch includes changes needed to output metadata that can be plotted in a grafana dashboard. Example is attached.

Screen Shot 2020-01-29 at 3 22 53 PM

### CODE REVIEW

File: bin/pycbc_inspiral [hunk 1]
```diff
@@ -448,4 +448,13 @@ if opt.fftw_output_float_wisdom_file:
...
 
+# Output metadata for Grafana Dashboard
+metadata_logging_str = event_mgr.output_json_metadata()
+logging.info("Writing metadata")
+logging.info(metadata_logging_str)
+
 logging.info("Finished")
+
+# Output metadata for Grafana Dashboard
+metadata_logging_str = event_mgr.output_json_metadata()
```
Comment (ahnitz) [inline bin/pycbc_inspiral]: Why is this called a second time?

Comment (duncan-brown) [reply bin/pycbc_inspiral]: I think that's a typo.

**Comment by josh-willis** [comment]
I think this looks good to me, if Alex agrees (and we remove the duplicate call to `event_mgr.output_json_metadata()`)

**Comment by ahnitz** [comment]
@duncan-brown What's going on with this PR?

**Comment by duncan-brown** [comment]
@ahnitz it would be good to get this merged in. Steven has left for industry, so let me add this to the list of things to look at.

**Comment by spxiwh** [comment]
This has been inactive for 3 years now. Closing now, but reopen if needed.

**Pull request closed without merging**